tests/snmpsim/coriant.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.42229.6.22
```

Comment:
Whilst it can vary this is purely for testing so you should add what you have available, the idea is that we can then ensure other changes don't break detection